.travis.yml

```diff
@@ -42,7 +42,7 @@ env:
     - PANTS_SHA="release_1.2.0" TEST_SET=jvm-integration
 
 script:
-  - ./scripts/run-tests-ci.sh
+  - ./scripts/run-tests-ci.sh --test-junit-output-mode=ALL
```

Comment:
revert. you can use PANTS_TEST_JUNIT_OUTPUT_MODE=ALL in travis settings to do the same.

Reply:
done, left a comment to that effect on the commit